### PR TITLE
Add zh trans for docs/reference/glossary/host-aliases.md

### PR DESCRIPTION
**Pull request by ydcool**
Add zh trans for docs/reference/glossary/host-aliases.md

### CODE REVIEW

**Comment by sftim** [comment]
Please rebase this pull request against _master_ to bring in a work-around for issue #22222.

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+title: 主机别名
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/host-aliases.md:2]: 在这个上下文中就不要翻译了。

Comment (ydcool) [reply content/zh/docs/reference/glossary/host-aliases.md]: 👌

**Pull request merged**